build.gradle

```diff
@@ -19,7 +19,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.0'
+        classpath 'com.android.tools.build:gradle:2.3.1'
```

Comment:
Any specific reason for this change?

Reply:
Nothing special besides Android Studio prompting for it.